quickstarts/kit/custom-istio-enricher/istio-enricher/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>eclipse-jkube-sample-custom-enricher-istio</artifactId>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.eclipse.jkube.quickstarts.kit</groupId>
+    <artifactId>eclipse-jkube-sample-custom-enricher-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <name>Eclipse JKube :: Quickstarts :: Kit :: Custom Istio Enricher</name>
+  <description>Custom Istio Enricher which extends Eclipse JKube Enricher API and Fabric8 Istio Java Client to create Istio Gateway Manifest</description>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.eclipse.jkube</groupId>
+      <artifactId>jkube-kit-enricher-api</artifactId>
+      <version>${jkube.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.jkube</groupId>
```

Comment:
Something doesn't look right here. Looks like the result of the famous copy-paste+sed pattern.